corehq/motech/openmrs/dbaccessors.py

```diff
@@ -0,0 +1,13 @@
+from corehq.motech.openmrs.models import OpenmrsImporter
+from corehq.util.quickcache import quickcache
+
+
+@quickcache(['domain_name'])
+def get_openmrs_importers_by_domain(domain_name):
+    results = OpenmrsImporter.get_db().view(
+        'by_domain_doc_type_date/view',
+        key=[domain_name, 'OpenmrsImporter', None],
+        include_docs=True,
+        reduce=False,
+    ).all()
+    return [OpenmrsImporter.wrap(result['doc']) for result in results]
```

Comment:
If you use `OpenmrsImporter.view(...)` (without the `get_db()`) then the wrapping is done for you